CONTRIBUTING.adoc

```diff
@@ -0,0 +1,55 @@
+= Contributing to Chaos Monkey for Spring Boot
+Thank you for willing to contribute! Every bit of help is highly welcome. 
+
+This document should help you get started if you want to contribute or hack on the code.
+
+**This is your first contribution to an open source project?** You can learn how to fork a project and how to create your first Pull Request from the excellent https://github.com/firstcontributions/first-contributions/blob/master/README.md[First Contributions] initiative.
+
+If you are unsure where to start, please head over to our https://github.com/codecentric/chaos-monkey-spring-boot/issues?q=is%3Aissue+is%3Aopen+label%3Agood-first-issue[Issue Tracker] where we have marked good beginner friendly issues. 
+
+
+== Using GitHub Issues
+We are using GitHub issues to track bugs, improvements and new features. If you have a general question on how to use Chaos Monkey for Spring Boot, please ask on Stack Overflow.
```

Comment:
Linking to our newly created tag as soon, as we created it? :D

Reply:
Hmm unfortunately I guess I don't understand what you mean. I would link to all open issues in the tracker, is that okay?

Reply:
👍